Tests/LoggingTests/LoggingTest.swift

```diff
@@ -1053,6 +1053,11 @@ class LoggingTest: XCTestCase {
         LoggingSystem.bootstrap(StreamLogHandler.standardOutput, metadataProvider: .exampleMetadataProvider)
         LoggingSystem.bootstrap(StreamLogHandler.standardError, metadataProvider: .exampleMetadataProvider)
     }
+    
+    func testLoggerSize() {
+        let expectedSize = MemoryLayout<Logger.Storage>.size + MemoryLayout<String>.size
```

Comment:
```suggestion
        let expectedSize = MemoryLayout<UnsafeRawPointer>.size
```

We need the `Logger` to become pointer-sized. So we should pull the `String` into the `Storage` too